source/guide_grocy.rst

```diff
@@ -0,0 +1,144 @@
+.. highlight:: console
+
+.. author:: Jonas <https://github.com/jfowl>
+
+.. tag:: lang-php
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/grocy.svg
+      :align: center
+
+####
+Grocy
+####
+
+.. tag_list::
+
+Grocy_ is a web-based self-hosted groceries & household management solution for your home.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`Domains <web-domains>`
+
+License
+=======
+
+Grocy is released under the `MIT license`_.
+
+Prerequisites
+=============
+
+Your URL needs to be setup:
+
+.. include:: includes/web-domain-list.rst
+
+Installation
+============
+
+Step 1
+-----
+
+Download grocy to your home directory.
+
+::
+
+ [isabell@stardust isabell]$ wget https://releases.grocy.info/latest -O grocy_latest.zip
+ […]
+ Saving to: ‘grocy_latest.zip’
+ 
+ grocy_latest.zip        100%[===============================>]  54,83M  12,6MB/s    in 4,4s
+
+ 2020-08-26 12:09:31 (12,5 MB/s) - ‘grocy_latest.zip’ saved [57489683/57489683]
+ [isabell@stardust isabell]$ 
+
+Step 2
+------
+
+Unpack the downloaded grocy release
+
+::
+
+ [isabell@stardust isabell]$ unzip -q grocy_latest.zip -d grocy
+ [isabell@stardust isabell]$ rm grocy_latest.zip
+ [isabell@stardust isabell]$ 
+
+
+Step 3
+------
+
+If you want to only host grocy on your Uberspace (best practice since Uberspace 7) you should link your
+DocumentRoot to grocy's public folder:
+
+::
+
+ [isabell@stardust isabell]$ cd /var/www/virtual/$USER/
+ [isabell@stardust isabell]$ rm -r html
```

Comment:
```suggestion
 [isabell@stardust isabell]$ rmdir html
```

Guides should assume a fresh uberspace, which means an empty `html` dir. Please change this to `rmdir`, so it fails if there is something in the directory. Users can then still choose to delete the contents on their own.